app/web/features/search/SearchPage.tsx

```diff
@@ -198,47 +160,63 @@ export default function SearchPage({
           >
             <SearchResultsList
               searchType={searchType}
-              setSearchType={setSearchType}
               locationResult={locationResult}
-              setLocationResult={setLocationResult}
               queryName={queryName}
-              setQueryName={setQueryName}
               results={data}
               error={errorMessage}
               hasNext={hasNextPage}
               selectedResult={selectedResult}
-              setSelectedResult={setSelectedResult}
               isLoading={isLoading || isFetching}
+              setSearchType={setSearchType}
+              setLocationResult={setLocationResult}
+              setQueryName={setQueryName}
+              setSelectedResult={(selectedResultParam) =>
+                setSelectedResult(selectedResultParam)
+              }
             />
           </Collapse>
         </Hidden>
         <FilterDialog
           isOpen={isFiltersOpen}
           queryName={queryName}
-          setQueryName={setQueryName}
-          onClose={() => setIsFiltersOpen(false)}
-          setLocationResult={setLocationResult}
           lastActiveFilter={lastActiveFilter}
-          setLastActiveFilter={setLastActiveFilter}
           hostingStatusFilter={hostingStatusFilter}
-          setHostingStatusFilter={setHostingStatusFilter}
           completeProfileFilter={completeProfileFilter}
-          setCompleteProfileFilter={setCompleteProfileFilter}
           numberOfGuestFilter={numberOfGuestFilter}
-          setNumberOfGuestFilter={setNumberOfGuestFilter}
+          setQueryName={(queryNameParam) => setQueryName(queryNameParam)}
+          onClose={() => setIsFiltersOpen(false)}
+          setLocationResult={(locationResultParam) =>
+            setLocationResult(locationResultParam)
+          }
+          setLastActiveFilter={(lastActiveParam) =>
+            setLastActiveFilter(lastActiveParam)
+          }
+          setHostingStatusFilter={(hostingStatusParam) =>
+            setHostingStatusFilter(hostingStatusParam)
+          }
+          setCompleteProfileFilter={(completeProfilesParam) =>
+            setCompleteProfileFilter(completeProfilesParam)
+          }
+          setNumberOfGuestFilter={(numberOfGuestsParam) =>
+            setNumberOfGuestFilter(numberOfGuestsParam)
+          }
```

Comment:
Except for the `onClose` prop, all of the other updates here are equivalent to passing the function reference directly to the prop since you're not doing anything additional, so this seems unnecessary to me until you need to do more work alongside the setter as it becomes more verbose to read 🤔